core/src/main/java/org/jdbi/v3/core/Handle.java

```diff
@@ -550,8 +550,8 @@ void setExtensionMethod(ExtensionMethod extensionMethod) {
         this.extensionMethod.set(extensionMethod);
     }
 
-    void setExtensionMethodThreadLocal(ThreadLocal<ExtensionMethod> extensionMethod) {
-        this.extensionMethod = requireNonNull(extensionMethod);
+    void setExtensionMethodThreadLocal(ThreadLocal<ExtensionMethod> extensionMethodThreadLocal) {
+        this.extensionMethod = requireNonNull(extensionMethodThreadLocal);
```

Comment:
Can we disable this rule on setters? This is a pretty common idiom.

Reply:
the rule is disabled on setters, but the param name must match the setter name

Reply:
http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.html

Reply:
This is package private so we can rename either the field or the setter so they match